nameko/rpc.py

```diff
@@ -351,6 +359,11 @@ def _call(self, *args, **kwargs):
             transport_options={'confirm_publish': True},
         )
 
+        serializer = (
+            container.config.get(SERIALIZER_CONFIG_KEY) or
+            DEFAULT_SERIALIZER
+        )
```

Comment:
Should probably use ``config.get(SERIALIZER_CONFIG_KEY, DEFAULT_SERIALIZER)`` as elsewhere